TestSuite/utils.py

```diff
@@ -1,55 +1,28 @@
 class IsEqualFunctions:
     @staticmethod
-    def is_dicts_equal(dict1: dict, dict2: dict):
-        if not isinstance(dict1, dict) or not isinstance(dict2, dict):
-            return False
-
-        if sorted(list(dict1.keys())) != sorted(list(dict2.keys())):
-            return False
-
-        for key, dict1_value in dict1.items():
-            dict2_value = dict2.get(key)
-
-            if not isinstance(dict2_value, type(dict1_value)):
-                return False
+    def is_dicts_equal(dict1: dict, dict2: dict, simple_comparison: bool):
```

Comment:
I would first check whether `dict1` and `dict2` are instances of `dict`
Or will the `: dict` notation throw a runtime exception?

Reply:
i'll add this test, as the notation doesn't affect runtime, thanks.

Reply:
see updated function